curriculum_algorithm/models.py

```diff
@@ -0,0 +1,122 @@
+from django.db import models
+
+POSITION_CHOICES = [
+    ('0', 'Any'),
+    ('1', 'Fall'),
+    ('2', 'Spring'),
+]
+
+
+class Course(models.Model):
+    course_number = models.CharField(max_length=8, unique=True)
+    season = models.CharField(max_length=1, choices=POSITION_CHOICES)
+    credit_hours = models.IntegerField(default=0)
+    laboratory = models.ForeignKey('Course', null=True, on_delete=models.SET_NULL, related_name='main_course')
+
+    def __str__(self):
+        return self.course_number
+
+
+class Curriculum(models.Model):
+    name = models.CharField(max_length=50, unique=True)
```

Comment:
In the class Curriculum of the initial design there was only the "department" field, but now you added "name". What's the difference between "name" and "department" in this case?